spec/services/journal_row_updater_spec.rb

```diff
@@ -17,4 +17,35 @@
     end
   end
 
+  describe "when the order detail is on a failed journal" do
+    let(:user) { FactoryGirl.create(:user) }
+    let(:facility) { FactoryGirl.create(:setup_facility) }
+    let(:account) { FactoryGirl.create(:setup_account, owner: user) }
+    # Original costs: actual: 20, subsidy: 10
```

Comment:
It's a pain. No easy way to inject it into `place_and_complete_item_order`. That's why I ended up and just did the comment.